system-tests/__snapshots__/webpack_dev_server_fresh_spec.ts.js

```diff
@@ -404,12 +404,12 @@ https://on.cypress.io/uncaught-exception-from-application
 
   (Screenshots)
 
-  -  /XXX/XXX/XXX/cypress/screenshots/Errors.cy.jsx/Errors -- error on mount (failed)     (1280x581)
+  -  /XXX/XXX/XXX/cypress/screenshots/Errors.cy.jsx/Errors -- error on mount (failed)     (1280x633)
      .png                                                                                           
-  -  /XXX/XXX/XXX/cypress/screenshots/Errors.cy.jsx/Errors -- sync error (failed).png     (1280x581)
-  -  /XXX/XXX/XXX/cypress/screenshots/Errors.cy.jsx/Errors -- async error (failed).pn     (1280x581)
+  -  /XXX/XXX/XXX/cypress/screenshots/Errors.cy.jsx/Errors -- sync error (failed).png     (1280x633)
+  -  /XXX/XXX/XXX/cypress/screenshots/Errors.cy.jsx/Errors -- async error (failed).pn     (1280x633)
```

Comment:
what was the main reason the height changed here?

Reply:
Passing `--disable-infobars' sets it back to what it was before.